superset/charts/api.py

```diff
@@ -709,3 +714,62 @@ def thumbnail(
         return Response(
             FileWrapper(screenshot), mimetype="image/png", direct_passthrough=True
         )
+
+    @expose("/export/", methods=["GET"])
+    @protect()
+    @safe
+    @statsd_metrics
+    @rison(get_export_ids_schema)
```

Comment:
Rison!! btw thks for creating this: https://github.com/betodealmeida/python-rison
tip: You can register the the rison spec on OpenApi like this: https://github.com/apache/incubator-superset/blob/master/superset/charts/api.py#L169

Then reference it like this: https://github.com/apache/incubator-superset/blob/master/superset/charts/api.py#L393

Reply:
Ah, thanks! #TIL!

I register the spec in https://github.com/apache/incubator-superset/pull/11373, I'll update the schemas in that PR.